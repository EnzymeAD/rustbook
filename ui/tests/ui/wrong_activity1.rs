#![feature(autodiff)]

#[autodiff(d_f1, Reverse, Active, Active)]
fn f1(x: &f64) -> f64 {
    x * x
}
