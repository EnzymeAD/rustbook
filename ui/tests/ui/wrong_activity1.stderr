error: Active can not be used for this type
 --> tests/ui/wrong_activity1.rs:4:10
  |
4 | fn f1(x: &f64) -> f64 {
  |          ^^^^
